### ISSUE TITLE
can't start after install

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo2js#82]
Here:

```
$ npm install -g eo2js

added 14 packages in 6s

2 packages are looking for funding
  run `npm fund` for details
$ eo2js
/opt/homebrew/bin/eo2js: line 1: syntax error near unexpected token `('
/opt/homebrew/bin/eo2js: line 1: `const {program} = require('commander')'
```

This is what I'm seeing in `/opt/homebrew/bin/eo2js`:

```
$ cat /opt/homebrew/bin/eo2js
const {program} = require('commander')
const transpile = require('./commands/transpile');
const link = require('./commands/link');
const version = require('./version');
const dataize = require('./commands/dataize');
const test = require('./commands/test');

program
  .name('eo2js')
  .usage('[options] command')
  .summary('EO to JS command line toolkit')
  .description('EOLANG to JavaScript command-line toolkit (' + version.what + ' built on ' + version.when + ')')
  .version(version.what, '-v, --version', 'Output the version number')
  .helpOption('-?, --help', 'Print this help information');

program
  .option('-t, --target <path>', 'Target directory with all generated files', '.eoc')
  .option('-p, --project <path>', 'Path to result JavaScript project', 'project')
  .option('-r, --resources <path>', 'Path to the resources', 'src/resources')
  .option('-d, --dependency <path>', 'Path to local eo2js-runtime dependency')
  .option('-f, --foreign <path>', 'Path to foreign tojos', 'eo-foreign.json')
  .option('--alone', 'Just run a single command without dependencies')
  // .option('--hash <hex>', 'Hash in objectionary/home to compile against', parser)
  // .option('--parser <version>', 'Set the version of EO parser to use', parser)
  // .option('--latest', 'Use the latest parser version from Maven Central')
  // .option('-b, --batch', 'Run in batch mode, suppress interactive messages')
  // .option('--no-color', 'Disable colorization of console messages')
  .option('--verbose', 'Print debug messages and full output of child processes');

```

I believe, the problem is that the shebang is missing: https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang

**Comment by maxonfjvipon** [comment]
@yegor256 I added shebang and released new version `0.0.7`. Please try it

**Comment by yegor256** [comment]
@maxonfjvipon now it works, thanks!
